sdk/eventhub/azure-eventhubs/azure/eventhub/aio/event_hubs_client_async.py

```diff
@@ -108,27 +109,76 @@ def _create_auth(self, username=None, password=None):
             eh_info = response.get_data()
             output = {}
             if eh_info:
-                output['name'] = eh_info[b'name'].decode('utf-8')
-                output['type'] = eh_info[b'type'].decode('utf-8')
-                output['created_at'] = datetime.datetime.fromtimestamp(float(eh_info[b'created_at'])/1000)
-                output['partition_count'] = eh_info[b'partition_count']
+                output['path'] = eh_info[b'name'].decode('utf-8')
+                output['created_at'] = datetime.datetime.utcfromtimestamp(float(eh_info[b'created_at'])/1000)
                 output['partition_ids'] = [p.decode('utf-8') for p in eh_info[b'partition_ids']]
             return output
         finally:
             await mgmt_client.close_async()
+    
+    async def get_partition_ids(self):
```

Comment:
Missing docstring...

Reply:
Yes. We have a separate issue to add/revise docstring, sample code and other documentation after this PR merged (code done). The API was constantly changing during the course of cross-language reference and arch board review in the past weeks.